src/streaming/controllers/PlaybackController.js

```diff
@@ -267,7 +267,7 @@ function PlaybackController() {
             if (playbackRate !== currentRate) {
                 logger.info('Starting live catchup mechanism. Setting playback rate to', playbackRate);
                 originalPlaybackRate = currentRate;
-                videoModel.getElement().playbackRate = playbackRate;
+                videoModel.setPlaybackRate(playbackRate);
```

Comment:
setPlaybackRate method defined in VideoModel is not exported. Currently it is defined as an internal function.

Reply:
You're right...I 've just added public visibility to this function.